### DISCUSSION TITLE
How to stop filament from changing my labels ?

### DISCUSSION THREAD

**Discussion by quasiperfect** [filamentphp/filament#9284] [Help]
hi

how can i stop filament from changing my labels ?

For example
![image](https://github.com/filamentphp/filament/assets/287513/d8c8cc9f-a8f9-44d8-892d-7a285315900b)

i expect to have the string : E-mailuri blocate

**Comment by VincvanAs** [comment]
Could you share some code please

**Reply by quasiperfect** [reply]
@VincvanAs i don't do anything special

in the resource class i have the usual functions

    public static function getModelLabel(): string
    {
        return __('manager-resources.coupon_ban.model.label');
    }

    public static function getPluralModelLabel(): string
    {
        return __('manager-resources.coupon_ban.model.plural_label');
    }

and in my language file

    'coupon_ban.model.label' => 'E-mailuri blocate',
    'coupon_ban.model.plural_label' => 'E-mailuri blocate',

**Reply by VincvanAs** [reply]
Could you try to direct output the translation key? 

Route::get('/test-translation', function() { return __('manager-resources.coupon_ban.model.label'); });

To see if the - is there?

**Reply by quasiperfect** [reply]
yes. i did a test and it returns 'E-mailuri blocate' as it should

**Reply by quasiperfect** [reply]
anyone ? i even created a new repo and the problem persist. my filament is up to date

**Comment by quasiperfect** [comment]
after checking for a bit the bread crumbs use `Str::headline(static::getPluralModelLabel())` that breaks my strings
